adds/supervisord.conf

```diff
@@ -0,0 +1,5 @@
+[supervisord]
+nodaemon = true
+
+[include]
+files = conf/*.conf
```

Comment:
I don't know supervisior. The messages produced by turnserver and matrix are printed onto console with supervisor?